### PR TITLE
Create aliSabry.json

### PR DESCRIPTION
**Pull request by ali-sabry**
To make our job easier, please spend time to review your application before submitting. 

#### Add a x between [] if you meet that requirement example `[x]`
- [ ] You're not using Vercel or Netlify.
- [x] You have completed your website, there's no type of placeholder at the website. **This requirement can be raised if the domain you're registering is for emails**
  - Link to website: 
- [x] The website is reachable.
- [x] The CNAME record doesn't contain any slash.
- [x] There's enough information at the `owner` field.
   - You should have your email presented. If you don't want to share email, you can leave email an empty string (`""`) and add any other social such as Discord/Twitter/etc.
 
 #### Feel free to join our discord server for more updates or any help related to the service :D - https://discord.gg/PZCGHz4RhQ

### CODE REVIEW

File: domains/alisabry.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+"repo": "https://alisabry.netlify.app",
+ "owner": {
+    "username": "ali-sabry",
+    "email": "alisabry.official@gmail.com"
+  },
+  "record": {
+    "CNAME": "alisabry.is-a.dev"
```
Comment (WilliamDavidHarrison) [inline domains/alisabry.json:8]: You cannot use this as your CNAME.

**Review by wdhdev — CHANGES_REQUESTED** [review]

**Review by wdhdev — CHANGES_REQUESTED** [review]
Invalid CNAME

File: domains/alisabry.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
       },
 
       "record": {
-        "URL": "https://alisabry.is-a.dev"
+        "URL": "https://alisabryofficial.is-a.dev"
```
Comment (WilliamDavidHarrison) [inline domains/alisabry.json:11]: Invalid URL

**Review by wdhdev — CHANGES_REQUESTED** [review]

**Review by wdhdev — CHANGES_REQUESTED** [review]
You cannot redirect to an is-a.dev subdomain.

**Review by wdhdev — APPROVED** [review]

**Review by MaskDuck — APPROVED** [review]

**Pull request merged**